examples/misc/raptor.py

```diff
@@ -8,6 +37,8 @@
 
 from radical.pilot import PythonTask
 
+logger = ru.Logger('radical.pilot')
```

Comment:
Oh! This is ultimately a wrapper for `logging.Logger`, right? Is this generally how client code is supposed to use the `logging` module when RCT is present?

Is there a way to override the absence of the `RADICAL_PILOT_LOG_LVL` environment variable programmatically?

Reply:
Is the terminal output suppressed for `'radical.pilot'` even with RADICAL_PILOT_LOG_LVL=DEBUG? Should I attach a regular character stream handler from the `logging` module if I want to see these messages on the console?

Reply:
We generally avoid to log to stdout/stderr.  If you want to force the logger to do so, you can set `RADICAL_LOG_TGT=stdout` (see [here](https://github.com/radical-cybertools/radical.utils/blob/devel/src/radical/utils/logger.py#L192) (*))

(*) the RU documentation is not building correctly for readthedocs right now, looking into it.

Reply:
> We generally avoid to log to stdout/stderr. If you want to force the logger to do so

I do, but I see that `Reporter` is used in other examples. Would that be more appropriate, do you think?

> the RU documentation is not building correctly for readthedocs right now, looking into it.

See also https://github.com/radical-cybertools/radical.utils/issues/359